### DISCUSSION TITLE
Change Startup Page

### DISCUSSION THREAD

**Discussion by vmontro** [lepoco/wpfui#609] [Q&A]
hello, how can i change the startup page other than DashboardPage ?
Thanks

**Comment by Veritera** [comment]
A little bit more info would be nice.

**Comment by rasyidf** [comment]
Assuming you're using the WPF UI template, it should be within the `\Services\ApplicationHostService.cs` file, you can change it by selecting the desired page here

```cs
private async Task HandleActivationAsync()
{
    if (!Application.Current.Windows.OfType<MainWindow>().Any())
    {
        _navigationWindow = (
            _serviceProvider.GetService(typeof(INavigationWindow)) as INavigationWindow
        )!;
        _navigationWindow!.ShowWindow();

        _navigationWindow.Navigate(typeof(Views.Pages.DashboardPage)); // <- Change Here
    }

    await Task.CompletedTask;
}
```

**Comment by LCYandCjjFoever** [comment]
How to use this template in the framework framework

**Comment by LCYandCjjFoever** [comment]
Does the project support running under the framework version
